### ISSUE TITLE
Firefox: NoScript always blocks scripts included via script src over file protocol

### ISSUE THREAD

**Issue by pal1000** [hackademix/noscript#115]
I don't know since when this regressed. but it was months ago for sure and I am sorry for reporting it so late.

Testcase
Part 1 : test.html
````
<script src='test.js'>
</script>
````
Part 2 :  test.js
````
document.write('Noscript bug that always blocks scripts included via &lt;script src=...&gt;&lt;/script&gt; over file protocol has been fixed');
````
Final step :  Open test.html and allow both test.html and test.js
**Result** :  Even after allowing both script sources page is blank. Even disabling restrictions on current tab or globally doesn't help.
**Expected result** : Intended message should appear after the 2 script sources are authorized.

System
--------
Windows 10 x64
Firefox  70.0.1 x64
Noscript 11.0.8rc1

**Comment by hackademix** [comment]
It works for me, with your Firefox & NoScript version (albeit on Linux, but it should not matter).
Could you try on a clean profile with just NoScript installed in its default configuration?

**Comment by pal1000** [comment]
Sometimes it works for me too if I refresh a few times and stops working if I continue refreshing. Looks like one of those annoying race condition bugs.

Tried clean profile with default settings, it made no difference.

**Comment by hackademix** [comment]
OK, managed to reproduce. It happens only for scripts included before the body. Checking what can be done to fix it.

**Comment by hackademix** [comment]
Should be fixed in [11.0.9rc2](https://github.com/hackademix/noscript/releases/tag/11.0.9rc2), thanks.
